src/Base/Models/PortalLinkGenerator.php

```diff
@@ -108,9 +110,15 @@ private function appendSubTheme(): self
             return $this;
         }
 
-        $subtheme = $this->post->getConnected('pdc-item_to_pdc-subcategory') ?? '';
+        $theme = $this->post->getConnected('pdc-item_to_pdc-category');
+        if (! $theme instanceof WP_Post) {
```

Comment:
`getConnected()` geeft een WP_Post terug of null. Dus je kunt de code veel eenvoudiger en duidelijker maken:

```php
        $subtheme = $this->post->getConnected('pdc-item_to_pdc-subcategory');
        $this->updatePortalURL($subtheme === null ? 'subthema' : $subtheme->post_name );
        return $this;
 
```

Reply:
Ik heb hem liever zo:
`$this->updatePortalURL($subtheme instanceof WP_Post ? $subtheme->post_name : 'subthema' );`